pom.xml

```diff
@@ -358,6 +358,24 @@
 					<!-- downgrade to fix Maven build with Java 11 -->
 					<!-- see https://bugs.openjdk.java.net/browse/JDK-8212233 -->
 					<source>8</source>
+                    <!-- https://stackoverflow.com/questions/38707030/maven-javadoc-plugin-and-inheritdoc-for-java-api-core-classes -->
```

Comment:
You seem to use whitespaces for indentation, rest of the document uses tabs—as it should be. 😉

Reply:
IntelliJ, always struggling with eclipse style 😉